ansible/roles/test/tasks/warm-reboot-fib.yml

```diff
@@ -0,0 +1,73 @@
+#--------------------------------------------------
+# Run FIB Warm Reboot test and Perform log analysis.
+#--------------------------------------------------
+# Perform prechecks
+- include: shared-fib.yml
+
+- debug : msg="Start FIB_WARM Test"
+
+# set ptf fact for this test
+- set_fact:
+    ptf_test_name: FIB test
+    ptf_test_dir: ptftests
+    ptf_test_path: fib_test.FibTest
+    ptf_platform: remote
+    ptf_platform_dir: ptftests
+    ptf_test_params:
+      - testbed_type='{{testbed_type}}'
+      - router_mac='{{ansible_Ethernet0['macaddress']}}'
+      - fib_info='/root/fib_info.txt'
+      - ipv4={{ipv4}}
+      - ipv6={{ipv6}}
+    ptf_extra_options: "--relax --debug info --log-file /tmp/fib_test.FibTest.ipv4.{{ipv4}}.ipv6.{{ipv6}}.{{lookup('pipe','date +%Y-%m-%d-%H:%M:%S')}}.log "
+
+- name: "Show ptf params PTF Test - {{ ptf_test_name }}"
+  debug: msg="ptf --test-dir {{ ptf_test_dir }} {{ ptf_test_path }} {% if ptf_qlen is defined %} --qlen={{ ptf_qlen }} {% endif %} {% if ptf_platform_dir is defined %} --platform-dir {{ ptf_platform_dir }} {% endif %} --platform {{ ptf_platform }} {% if ptf_test_params is defined %} -t \"{{ ptf_test_params | default([]) | join(';') }}\" {% endif %} {{ ptf_extra_options | default(\"\")}} --disable-vxlan --disable-geneve --disable-erspan --disable-mpls --disable-nvgre 2>&1"
+
+- name: Run PTF Traffic for Fib
+  async: 1000
```

Comment:
why this step cannot use ptf_runner?

Reply:
Async is not supported as of now for include module or for entire block. So have to pull it out of ptf runner.
Another way can be: Run this task in ptf_runner.yml async/sync based on a parameters from caller playbook. For this result of async task must be register in fixed variable say async_ptf_runner, so that caller playbook can wait for async task. [We may need this in future when more playbook will be written async way for warm reboot]. But as if now it is better to not update ptf_runner. Kindly let me know. Thx.